000458/FlatironInstitute/PSTH_neurosift_exploration.md

```diff
@@ -0,0 +1,39 @@
+# 000458 - PSTH Neurosift Exploration
+
+See [001_summarize_contents.ipynb](./001_summarize_contents.ipynb) for a summary of the contents of the NWB files in this Dandiset, including the trial information. You can use that to decide which NWB files to examine. Here's an example of one that targets the motor cortex:
+
+```
+Asset 7: sub-551397/sub-551397_ses-20210211_behavior+ecephys.nwb -- size: 15202.44 MB
+in vivo electrophysiology in a head-fixed mouse during cortical electrical microstimulation
+EEG and Neuropixels recording during wakefulness and isoflurane anesthesia
+single pulse electrical stimuli targeted to MOs (deep layers)
+  1800 trials
+  BEHAVIORAL EPOCHS
+    awake: 386 valid and not running
+    isoflurane: 480 valid and not running
+    recovery: 553 valid and not running
+  STIMULUS TYPES
+    electrical: 1103 valid and not running
+    visual: 316 valid and not running
+  STIMULUS DESCRIPTIONS
+    biphasic: 1103 valid and not running
+    white circle: 316 valid and not running
+  ESTIM TARGET REGIONS
+    MOs: 1103 valid and not running
+    n/a: 316 valid and not running
```

Comment:
FWIW, could have that script could have (optionally?) produced markdown with neurosift URLs pointing to those particular files/settings in neurosift to select the desired trials?  or URL would be a bit unruly to create and fragile (might change in the future)?

Reply:
Great idea. It's now markdown with links to NS. Not as fancy as the settings as you suggest, but it gets you to the file.